File: app/app/Model/Chat/Response/ChatRoomGetResponse.swift
struct ChatRoomGetResponse: Codable, Identifiable {

    let chatRoomId: Int64
    let targetId: Int64
    let nickname: String
    let profileUrl: String?
    let lastMessage: String?
    let lastMessageAt: String?
    let sortAt: String
    let unreadCount: Int

    var id: Int64 { chatRoomId }
}

File: server/src/main/kotlin/com/pidulgi/server/chat/dto/response/ChatRoomGetResponse.kt
package com.pidulgi.server.chat.dto.response

import java.time.LocalDateTime

data class ChatRoomGetResponse(

    val chatRoomId: Long,
    val targetId: Long,
    val nickname: String,
    val profileUrl: String?,
    val lastMessage: String?,
    val lastMessageAt: LocalDateTime?,
    val sortAt: LocalDateTime,
    val unreadCount: Int,
)
